SEED-platform/seed
# 2320 apiv3 frontend vstlpe

#### Any background context you want to provide?
Part of the effort to convert the frontend to use API v3

#### What's this PR do?
See attached issue.

#### How should this be manually tested?
Verify checks in attached issue are warranted.

Also check the following:
Confirm that there are no usages of the scenarios endpoint outside the scope of swagger?

#### What are the relevant tickets?
#2320
